lib/linux_admin/network_interface.rb

```diff
@@ -112,7 +114,11 @@ def netmask6(scope = :global)
     #
     # @return [String] IPv4 gateway address
     def gateway
-      @network_conf[:gateway]
+      @network_conf[:gateway4]
+    end
+
```

Comment:
Yeah, the single/multiple interface thing will make all of this much more difficult. :worried: I'm not sure how we deal with that currently ...